roles/ceph-mon/tasks/docker/pre_requisite.yml

```diff
@@ -15,14 +15,33 @@
     state: present
   when: ansible_os_family == 'Debian'
 
+- name: install epel-release on redhat
+  yum:
+    name: epel-release
```

Comment:
is it sufficient to get epel-release?
don't you need something like: `rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm`?